src/bidders/prebid/index.js

```diff
@@ -163,6 +158,31 @@ export class Prebid extends BaseBidder {
 		);
 	}
 
+	refreshBids(slotAlias) {
+		if (!this.refreshSlots.includes(slotAlias)) {
+			this.refreshSlots.push(slotAlias);
+		}
+
+		if (!this.refreshEnabled) {
+			this.refreshEnabled = true;
+
+			window.pbjs.onEvent('bidWon', (winningBid) => {
+				if (this.refreshSlots.includes(winningBid.adUnitCode)) {
+					const adUnitRefresh = this.adUnits.filter(
+						adUnit => (
+							adUnit.code === winningBid.adUnitCode &&
+							adUnit.bids &&
+							adUnit.bids[0] &&
```

Comment:
It's always one-element array, right?

Reply:
Not always. For some bidders it's multi-element array, but every element has only different sizes, so it's enough to check first only.